.gitignore

```diff
@@ -0,0 +1,48 @@
+npm-debug.log
+.vagrant
+*~
+*#
+.#*
+\#*#
+.*.sw[a-z]
+*.un~
+/cookbooks
+.DS_Store
+*.py[co]
+*.egg
+PKG-INFO
+build/
+target/
+
+# Bundler
+.bundle/*
+
+# Coverage reports
+coverage
+
+.kitchen/
+.kitchen.local.yml
+
+*.iml
+.idea/
+
+/build
+.gradle
+
+# Cookbooks tarball
+lib/provisioner/worker/plugins/automators/chef_automator/chef_automator/*.tar.gz
+lib/provisioner/worker/plugins/automators/shell_automator/*.tar.gz
```

Comment:
These directories either don't exist or the tarballs aren't created in these directories, anymore.